### PR TITLE
Operator tests fix 

### PR DESCRIPTION
**Pull request by eliasalveszup**
- Added validation on request to verify whether the specified namespace exists and is active
- FIxing 500 error when trying to access invalid github or gitlab chart url (changed to 400)

### CODE REVIEW

File: butler/src/app/v2/core/integrations/k8s/client.ts [hunk 1]
```diff
@@ -81,15 +83,47 @@ export class K8sClient {
...
+    } catch (error) {
+      if (error.response.body.code == 404){
+        return {
+          body: {},
+          response: error.response
+        }
+      }
+      this.consoleLoggerService.error('ERROR:SEARCH_FOR_NAMESPACE', error)
+      throw new InternalServerErrorException({
+        errors: [
```
Comment (leandrorodrigueszup) [inline butler/src/app/v2/core/integrations/k8s/client.ts:105]: All these details, shouldn't be on borders of the application? It seems to be a title to much put it in all error handling!!

Comment (eliasalveszup) [reply butler/src/app/v2/core/integrations/k8s/client.ts:105]: I agree. There's a lot of places with a code like this. It's necessary to refactor the error handling, encapsulating the error building and providing a simple way to throw it. Requires a rebuild

File: butler/src/app/v2/core/integrations/github/github-repository.ts [hunk 2]
```diff
@@ -81,6 +81,25 @@ export class GitHubRepository implements Repository {
...
+                timestamp: Date.now()
+              },
+              source: {
+                pointer: 'components.helmRepository'
+              },
+              status: `${error.response.status}`
+            }
+          ]
+        })
+      })
```
Comment (lucasbfernandes) [inline butler/src/app/v2/core/integrations/github/github-repository.ts:103]: You are forcing this error to be a BadRequest. What if the request failed for another reason?

**Review by lucasbfernandes — CHANGES_REQUESTED** [review]

File: butler/src/app/v2/core/integrations/github/github-repository.ts (hunk 2 above)
Comment (eliasalveszup) [reply butler/src/app/v2/core/integrations/github/github-repository.ts]: Done

**Review by lucasbfernandes — APPROVED** [review]

**Pull request merged**